deeptime/util/callbacks.py

```diff
@@ -58,6 +59,8 @@ def __init__(self, progress, description=None, total=None):
 
     def __call__(self, inc=1, *args, **kw):
         self.progress_bar.update(inc)
+        if 'error' in kw:
+            self.progress_bar.set_description("{} - [inc: {:.1e}]".format(self.description, kw.get('error')))
```

Comment:
this feels like it should go into the implementing subclass

Reply:
Well this is a super specific call to `set_description` with a certain formatting and all. A general superclass isn't really the place for that :slightly_smiling_face:

Reply:
ugh I know D: ok I made it better

Reply:
thanks!